lib/pluck_each.rb

```diff
@@ -16,10 +16,13 @@ def pluck_each(*args)
 
     def pluck_in_batches(*column_names)
       options = column_names.extract_options!
+      string_column_names = column_names.map(&:to_s)
```

Comment:
Since line 18 modifies the column names hash, it's probably fine (and more efficient) to use `map!` here.